docs/DeveloperGuide.md

```diff
@@ -263,8 +263,9 @@ _{Explain here how the data archiving feature will be implemented}_
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
-
+**Value proposition**: Empower students with the ability to make sensible financial decisions in their budgeting. 
```

Comment:
I'm not sure if this value proposition is too big or not.
However, if we can have multiple value propositions, perhaps it would be better to display it in bullet format?

Reply:
I have put it as bullet points for now, if there's a need to change the formatting or cut the length, we can do it again in the future.